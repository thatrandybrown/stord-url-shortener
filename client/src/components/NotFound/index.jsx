import Error from "../Error";

const NotFound = () => <Error heading="404" detail="Page Not Found!" />;

export default NotFound;
