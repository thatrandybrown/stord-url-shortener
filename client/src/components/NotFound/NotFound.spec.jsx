import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import NotFound from "./";

test("renders the NotFound Component", () => {
  const { container, getByText } = render(<NotFound />, {
    wrapper: MemoryRouter,
  });
  expect(container.firstChild).toMatchSnapshot();
  expect(getByText("404")).toBeInTheDocument();
  expect(getByText("Page Not Found!")).toBeInTheDocument();
});
