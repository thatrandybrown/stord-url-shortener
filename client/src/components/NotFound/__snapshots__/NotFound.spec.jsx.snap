// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the NotFound Component 1`] = `
<section
  class="error-section"
>
  <h2
    style="font-size: 10rem;"
  >
    404
  </h2>
  <p>
    Page Not Found!
  </p>
  <a
    href="/"
  >
    Navigate Home
  </a>
</section>
`;
